data/controller.xql

```diff
@@ -205,6 +205,12 @@ else if($exist:path = '/create' and request:get-method() eq 'POST') then
         if(request:get-header('Accept') eq 'application/json' and $backend-response instance of map(*))
         then local:stream-json(map:remove($backend-response, 'document-node'), $backend-response?code)
         else local:redirect-to-main-page()
+        
+else if($exist:path = '/list' and request:get-method() eq 'GET') then 
+    let $fileNames := crud:list($config:data-root)
+    let $json := serialize($fileNames, map { "method": "json" })
+    return
+        response:stream($json, "media-type=application/json")
```

Comment:
I propose to remove the `$database` parameter from `crud:list` because the MerMEId is (currently) only able to work with one shallow collection of MEI files.
```suggestion
(:~
 : list files endpoint 
 :
 :)
else if($exist:path = '/list' and request:get-method() eq 'GET') then 
    let $fileNames := crud:list()
    let $json := serialize($fileNames, map { "method": "json" })
    return
        response:stream($json, "media-type=application/json")
```